### DISCUSSION TITLE
invalid option: --with-orc

### DISCUSSION THREAD

**Discussion by ECJ222** [Homebrew/discussions#4062] [Writing Formulae/Casks]
An error occurs when trying to install Gstreamer's additional plugins on macOS Monterey 12.4 terminal.

```
brew install gstreamer gst-plugins-base gst-plugins-good \
      gst-plugins-bad gst-plugins-ugly gst-libav gst-rtsp-server \
      gst-editing-services --with-orc --with-libogg --with-opus \
      --with-pango --with-theora --with-libvorbis --with-libvpx \
      --enable-gtk3
```

```
Error: invalid option: --with-orc
```

**Comment by gromgit** [comment]
`--with-XYZ` build options have been removed from Homebrew core formulae for **years**. Try `brew install` without all those options.

**Reply by ECJ222** [reply]
Thanks, @gromgit.

That works, but do you know another way I could install those plugins/libraries?

**Reply by alebcay** [reply]
Can you `brew install gst-plugins-base` and then run `gst-inspect-1.0` and check that the plugins you're expecting are present? `gst-plugins-base` should already include integrations with a number of other libraries, including `opus`, `orc`, `libogg`, `pango`, and `theora` (i.e. the default is currently to build and include everything).

**Reply by ECJ222** [reply]
It seems `opus`, `pango`, and `theora` are already installed. However, I wasn't able to find `orc` and `libogg` @alebcay.
